### ISSUE TITLE
Scrcpy not working wirelessly

### ISSUE THREAD

**Issue by s3hMC** [Genymobile/scrcpy#1561]
I'm pretty sure I did everything right, but on my Windows device (which is connected to Ethernet), when I enter the command `scrcpy` the command-line just closes without anything happening.

**Comment by rom1v** [comment]
> Scrcpy not working wirelessly

> on my Windows device (which is connected to Ethernet)

So does it work over USB?

What is the result of `adb devices`?

**Comment by s3hMC** [comment]
Yes, it works over USB.

`adb devices` comes up with two devices. One is the IP of the device, and the other is a string with numbers and letters.

**Comment by rom1v** [comment]
OK. What happens when you execute:

```
scrcpy -s 192.168.0.1:5555
```

? (if the ip is 192.168.0.1)

**Comment by s3hMC** [comment]
```
adb: error: failed to get feature set: no devices/emulators found
ERROR: "adb push" returned with value 1
Press any key to continue...
```

**Comment by s3hMC** [comment]
```
adb: error: failed to get feature set: no devices/emulators found
ERROR: "adb push" returned with value 1
Press any key to continue...
```

**Comment by rom1v** [comment]
No, it's just that your device is not detected by `adb`. See https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#adb-issues

**Comment by AruArshad** [comment]
Any updates on this? Facing the same error with my Huawei P30 Pro after updating scrcpy to the latest version

**Comment by rom1v** [comment]
@s3hMC 

> adb: error: failed to get feature set: no devices/emulators found

If it's over wifi, then you did not `adb connect` correctly.

**Comment by zulfi0** [comment]
> Could it be because of my VPN?

what kind of vpn you use ? is it possible to use vpn like portmap ?
